src/components/pages/CaseTable/CaseTable.js

```diff
@@ -494,29 +497,79 @@ export default function CaseTable(props) {
     });
 
     return (
+      // <Plot
```

Comment:
Did you want to leave this code in for reference later? Or is it needed?

Reply:
Yup, this code is a reference for later, will be deleting it soon!

Reply:
Perfect! Thanks for clarifying!